integration_tests/src/expects.rs

```diff
@@ -42,6 +45,8 @@ impl Expect {
     pub fn market_activate_deals(
         from: ActorID,
         deals: Vec<DealID>,
+        // TODO: A more future-proof one would take the deals as a Vec<(DealID, ActorId)>
```

Comment:
Nit: remove the word "TODO". No-one's going to do it until they're forced to. Leaving a note here is great, but "TODO" doesn't add any information.

Reply:
Removed this.